tests/test_dataset.py

```diff
@@ -22,14 +23,13 @@
     's3://scaleapi-attachments/BDD/BDD/bdd100k/images/100k/train/89b42832-10d662f4.jpg',
 ]
 
-@pytest.fixture(scope='module')
+@pytest.fixture()
```

Comment:
Decided to do this so our different test functions have separate state. Otherwise, if one function causes the dataset to bug out somehow, everything else will fail from that.

Reply:
good catch!